temporal/t.rast.algebra/t.rast.algebra.html

```diff
@@ -561,6 +567,13 @@ <h3>Computation with time intervals with temporal topology relation</h3>
 C = if({equal}, B {#,contain} A > 1, (B {+,contain,l} A {-,equal,l} B) {equal,=/} (B {#,contain} A), A)
 </pre></div>
 
+<h3>Compute DOY for spatio-temporal conditions</h3>
+Compute the DOY for all maps from STRDS A where conditions are met at three 
+consecutive time intervals (e.g. temperature > 0):
+<div class="code"><pre>
+B = if(A>0.0&&A[-1]>0.0&&A[-2]>0.0, start_doy(A, -1), 0)"
```

Comment:
```suggestion
B = if(A > 0.0 && A[-1] > 0.0 && A[-2] > 0.0, start_doy(A, -1), 0)"
```
adding spaces for better readability